src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -96,6 +96,7 @@ public class ElasticsearchClient {
       )
   );
 
+  private final Boolean logSensitiveData;
```

Comment:
nit: This can be `boolean` unless you have a reason to be `Boolean`

Reply:
Thanks, done